lib/engine/corporation.rb

```diff
@@ -213,5 +214,15 @@ def transfer(ownable_type, to)
       ownables.clear
       transferred
     end
+
+    def closed?
+      @closed
+    end
+
+    def close!(bank)
```

Comment:
maybe here we sohuld remove the corporation from the share price

Reply:
why pass bank here? just set owner to be nil

Reply:
Have added a deletion of share price from stock market. Now I pass game instead. And set owner to nil.